source/subs_functions/subs_functions.bas

```diff
@@ -2833,11 +2833,11 @@ regid
 clearid
 id.n = "Cls"
 id.subfunc = 2
-id.callname = "sub_cls"
-id.args = 2
-id.arg = MKL$(LONGTYPE - ISPOINTER) + MKL$(ULONGTYPE - ISPOINTER)
-id.specialformat = "[?][,?]"
-id.hr_syntax = "CLS [method%] [, bgColor&]"
+id.callname = "sub_clsDest"
+id.args = 3
+id.arg = MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER) + MKL$(ULONGTYPE - ISPOINTER)

```

Comment:
It probably doesn't significantly matter, but I think it should probably be `LONG/ULONG/LONG` to keep the color type as an unsigned long. Your current change makes it `LONG/LONG/ULONG`.

Reply:
Swapped those around.  In libqb, I had the order correct; in the bas source, they were reversed.  Only 27 and a half places necessary to track such minor changes...  LOL, and I had to retweak 26 of those!  :D